.circleci/config.yml

```diff
@@ -314,29 +314,23 @@ jobs:
     - setup_remote_docker
     - run:
         command: |
-          # BUILD_FROM_REPO should ALWAYS be the main OSS Vault repo URL.
-          BUILD_FROM_REPO=git@github.com:hashicorp/vault.git
-          [ "$CIRCLE_REPOSITORY_URL" = "$BUILD_FROM_REPO" ] || {
-            echo "Not building website docker image for repo '$CIRCLE_REPOSITORY_URL' - we only build it for $BUILD_FROM_REPO."
-            exit 0
-          }
-          [ "$CIRCLE_BRANCH" = "master" ] || {
-            echo "Not building website docker image for branch '$CIRCLE_BRANCH' - we only build it for master."
-            exit 0
-          }
-          PACKAGE_LOCK_CHANGED="$( \
-            git diff --name-only $(git log --pretty=format:'%h' -n1 HEAD~1)...HEAD | \
-            grep -c website/package-lock.json; \
-          )" || true
-          [ $PACKAGE_LOCK_CHANGED -gt 0 ] || {
-            echo "Not building a new website docker image - dependencies have not changed."
-            exit 0
-          }
-          cd website/
-          docker build -t hashicorp/vault-website:$CIRCLE_SHA1 .
-          docker tag hashicorp/vault-website:$CIRCLE_SHA1 hashicorp/vault-website:latest
-          docker login -u $DOCKER_USER -p $DOCKER_PASS
-          docker push hashicorp/vault-website
+          # There is an edge case that would cause an issue here - if dependencies are updated to an exact copy
+          # of a previous version, for example if packge-lock.json is reverted, we need to manually push the new
+          # image to the "latest" tag
+          # Ignore job if running an enterprise build
+          IMAGE_TAG=$(cat website/Dockerfile website/package-lock.json | sha256sum | awk '{print $1;}')
+          echo "Using $IMAGE_TAG"
+          if  [ "$CIRCLE_REPOSITORY_URL" != "git@github.com:hashicorp/vault" ]; then
+            echo "Not Vault OSS Repo, not building website docker image"
+          elif curl https://hub.docker.com/v2/repositories/hashicorp/vault-website/tags/$IMAGE_TAG -fsL > /dev/null; then
```

Comment:
I wonder if we should worry about the case were the connection might get dropped sporadically (e.g. unstable connection right when this check is running).